apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt

```diff
@@ -219,6 +219,8 @@ class AppPreferences {
   }, settingsThemes)
   val themeOverrides = mkThemeOverridesPreference()
   val profileImageCornerRadius = mkFloatPreference(SHARED_PREFS_PROFILE_IMAGE_CORNER_RADIUS, 22.5f)
+  val chatItemRoundness = mkFloatPreference(SHARED_PREFS_CHAT_ITEM_ROUNDNESS, 0.75f)
+  val chatItemTail = mkBoolPreference(SHARED_PREFS_CHAT_ITEM_TAIL, true)
```

Comment:
enabling/disabling tail changes shapes of messages for some reason, not only showing/hiding the tail

Reply:
yes, there are a few small changes that can happen, in some cases we will render a roundrect with a different max value for the corners, the business logic was defined in IOS from https://github.com/simplex-chat/simplex-chat/blob/master/apps/ios/Shared/Views/Helpers/ChatItemClipShape.swift#L32 and translated here

Reply:
This is rather accidental because we wanted avoiding Bézier curves and just use squircles iOS does for rounded rectangles with "continuous" setting. So we don't really need to change the shape with/without tail, and the attempt in iOS was to minimize this change.